### DISCUSSION TITLE
beancount support

### DISCUSSION THREAD

**Discussion by ekarious** [ananthakumaran/paisa#78] [Ideas]
Just came across your project. Huge work here, congrats :)

I know it uses `.ledger` files and I wanted to know if this will be possible to make use of `.beancount` file in the future ?
That would be awesome.

**Comment by ananthakumaran** [comment]
It's definitely possible. Paisa only uses the ledger cli in a few places like editor, import, sync, etc. hledger integration is roughly 200 to 300 lines of code. That said, beancount might be more challenging than hledger, as it has slightly different syntax, etc. Also, the budget functionality depends on the virtual posting, which is not supported by beancount. It's not a high priority as of now, I might look into the details later.

**Reply by ekarious** [reply]
Thank you :) 
If you need some help, do not hesitate to ask me.  
I will try to look into it on my own when I have the time. At least for the basic feature working

**Reply by ananthakumaran** [reply]
I have added beancount support, available on 0.5.5.

**Reply by ekarious** [reply]
Thank you so much !
